### ISSUE TITLE
OTel collector agent is no longer logging all the files it has started watching

### ISSUE THREAD

**Issue by matthewmodestino** [signalfx/splunk-otel-collector#1433]
Hey Team, cross posting this from the helm chart repo:

https://github.com/signalfx/splunk-otel-collector-chart/issues/429#issue-1192166299

I have noticed recently the collector is no longer logging all the logs it finds with the `filelog` receiver. It seems to only show one log then doesnt info log any further logs its watching, despite sending more than one log to the backend. 

Can someone review whats up with the `filelog` info logging as they are helpful for troubleshooting. 

Before 0.47.0 helm chart version, you would see all the files it was actively monitoring:

```
2022-04-01T15:54:22.363Z	info	file/file.go:174	Started watching file	{"kind": "receiver", "name": "filelog", "operator_id": "file_input", "operator_type": "file_input", "path": "/var/log/pods/default_mattymo-splunk-otel-collector-agent-mmkv7_278b59aa-485d-45fe-9519-f3ad944b8122/migrate-checkpoint/1.log"}
2022-03-30T03:42:40.368Z	info	file/file.go:174	Started watching file	{"kind": "receiver", "name": "filelog", "operator_id": "file_input", "operator_type": "file_input", "path": "/var/log/pods/kube-system_calico-node-jv5kk_4f7cbaf6-a65e-4f4c-b5dc-2412eaa5a986/calico-node/0.log"}
2022-03-30T03:42:39.367Z	info	file/file.go:174	Started watching file	{"kind": "receiver", "name": "filelog", "operator_id": "file_input", "operator_type": "file_input", "path": "/var/log/pods/kube-system_calico-node-jv5kk_4f7cbaf6-a65e-4f4c-b5dc-2412eaa5a986/flexvol-driver/0.log"}
2022-03-30T03:42:36.768Z	info	file/file.go:174	Started watching file	{"kind": "receiver", "name": "filelog", "operator_id": "file_input", "operator_type": "file_input", "path": "/var/log/pods/kube-system_calico-node-jv5kk_4f7cbaf6-a65e-4f4c-b5dc-2412eaa5a986/upgrade-ipam/0.log"}
2022-03-30T03:41:39.767Z	info	file/file.go:174	Started watching file	{"kind": "receiver", "name": "filelog", "operator_id": "file_input", "operator_type": "file_input", "path": "/var/log/pods/kube-system_hostpath-provisioner-7764447d7c-qnh58_37180f7c-a795-4f13-b13c-8d657c1bb37f/hostpath-provisioner/1.log"}
2022-03-28T17:30:26.367Z	info	file/file.go:174	Started watching file	{"kind": "receiver", "name": "filelog", "operator_id": "file_input", "operator_type": "file_input", "path": "/var/log/pods/default_mattymo-splunk-otel-collector-agent-mmkv7_278b59aa-485d-45fe-9519-f3ad944b8122/migrate-checkpoint/0.log"}
```

Now on deploy, I only ever see one entry from "started watching file", despite there being more than one file being monitored and successfully forwarded

```
2022-04-04T18:29:53.360Z	info	file/file.go:174	Started watching file	{"kind": "receiver", "name": "filelog", "operator_id": "file_input", "operator_type": "file_input", "path": "/var/log/pods/default_mattymo-splunk-otel-collector-agent-hhpnn_6bfc4c1b-c608-4a87-b3fa-f85179c634e1/migrate-checkpoint/0.log"}
```

**Comment by atoulme** [comment]
I think this might have been fixed now. I don't reproduce with a simple filelog anymore. I am going to close this issue, please reopen if you can still reproduce with 0.68.0.

**Comment by rht6226** [comment]
Still facing the same issue. Even with 0.68.0 and 0.70.0 

![logs](https://user-images.githubusercontent.com/37151791/214495525-d463cda2-0209-4502-9f4f-f34801b3506e.PNG)
It starts watching the file but is not able to parse any logs.
I have the application running on the same cluster and I am trying to read the logs from /var/log/containers/ using filelog receiver. But no logs are getting parsed.

Although if I copy the same file somewhere else it is able to read logs from there

**Comment by atoulme** [comment]
We haven't released 0.70.0 yet. Are you using our distribution? The original issue stated that the problem was with the log itself not showing which files were being watched. It looks like your issue is with filelog not correctly ingesting data. Please open a separate issue and provide your configuration, thanks!
